doc/api-examples/c++/list_nodes.cpp

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Please change the license to `CC0-1.0` to align with other examples

Reply:
Here is the PR aligning other existing languages: https://github.com/containers/bluechi/pull/546